### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mikesalvia**
Adds how to re-run
Adds details that re-run is not automatic

### CODE REVIEW

**Comment by mikesalvia** [comment]
Adds documentation of #59

File: README.md [hunk 1]
```diff
@@ -54,6 +54,48 @@ export const config = {
...
+By design, this service does not automatically re-run failed tests.
+
+After WebDriver.IO has completed execution, if failures or unstable tests/scenarios are found a file will be at **rerunScriptPath** or by default `./rerun.sh` (see Configuration).
+
+### Conditional Re-run
+Every teams re-run needs will differ execution could be based on any number of factors, this is an example of how to accomplish a conditional re-run based on # of failures.
+
+##### attemptRerun.sh
+Executes `./rerun.sh` if less than 25 failures have been found in last execution of WebDriver.IO.
+```
```
Comment (christian-bromann) [inline README.md:67]:
```suggestion
```sh
```

File: README.md [hunk 2]
```diff
@@ -54,6 +54,48 @@ export const config = {
...
+	else
+    echo "Skipping re-run, expected < $MAX_TO_RERUN failures to qualify execution for re-run, got $NUMBER_OF_FAILURES failures."
+	fi
+else
+  echo "No [rerun.sh] file exits, skipping re-run!"
+fi
+```
+##### Bash Re-run Command
+Execute in shell
+```
```
Comment (christian-bromann) [inline README.md:85]:
```suggestion
```sh
```

File: README.md [hunk 3]
```diff
@@ -54,6 +54,48 @@ export const config = {
...
+. ./attemptRerun.sh
+```
+##### Integrate with NPM
+Add task in `package.json`
+```
```
Comment (christian-bromann) [inline README.md:90]:
```suggestion
```sh
```

File: README.md [hunk 4]
```diff
@@ -54,6 +54,48 @@ export const config = {
...
+"attempt-rerun": ". ./attemptRerun.sh"
+````
+##### NPM Re-run Command
+Execute in shell
+```
```
Comment (christian-bromann) [inline README.md:95]:
```suggestion
```sh
```

**Comment by mikesalvia** [comment]
@christian-bromann All suggestions have been commited.

**Review by christian-bromann — APPROVED** [review]
LGTM 👍

**Pull request merged**